README.md

```diff
@@ -73,30 +73,36 @@ docker pull miltondp/clustermatch_gene_expr
 The image only contains the conda environment with the code in this repository so, after pulling the image, you need to download the data as well:
```

Comment:
Exactly, the `environment/environment.yml` file specifies the python version (`3.9.*`). I would prefer to keep that info there, otherwise it might get outdated if that changes.